### ISSUE TITLE
Mesa Diretora

### ISSUE THREAD

**Issue by DTILegBG** [interlegis/sapl#876]
Cadastrando uma nova legislatura, e não cadastrando uma mesa para ela, clicando em institucional/mesa as mesas cadastradas não são mostradas, apenas a mensagem abaixo
![image](https://cloud.githubusercontent.com/assets/13314947/21223044/5b0ad0cc-c2ac-11e6-9e47-cbf3ab577422.png)

**Comment by LeandroRoberto** [comment]
simulei aqui... cadastrando a nova também... o que sugere?

**Comment by LeandroRoberto** [comment]
simulei aqui... cadastrando um nova também... o que falta é o usuário cadastrar a sessão legislativa... não fará sentido em nada mesmo, registrar uma Legislatura, e não registrar suas sessões legislativas... o que sugere para esse meio termo?

**Comment by DTILegBG** [comment]
talvez seja interessante aguardar e ver se vai ser implementado a 
questão da período de composição de mesa. Ai vai acabar mudando de 
qualquer forma.

Senão penso que tenha que vir para tela de seleção de legislatura.

Mas o comportamento está bem estranho, cadastrando a sessão legislativa 
para a legislatura e não cadastrando a mesa correspondente.



Em 15/12/2016 10:04, Leandro Roberto da Silva escreveu:

**Comment by edwardoliveira** [comment]
Sim, **antes** de cadastrar a Mesa deve-se cadastrar a Legislatura e ao menos uma Sessão Legislativa, até onde me lembro. O que vcs sugerem? Que já se insira as sessões ao se cadastrar a legislatura?

**Comment by andreagudo** [comment]
#493
